app/uk/gov/voa/propertylinking/errorhandler/CustomErrorHandler.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.voa.propertylinking.errorhandler
+
+import java.time.{Instant, LocalDateTime, ZoneId}
+
+import config.ApplicationConfig
+import javax.inject.Inject
+import play.api.Logger
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.Request
+import play.twirl.api.Html
+import uk.gov.hmrc.http.HeaderNames
+import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
+
+class CustomErrorHandler @Inject()(implicit val messagesApi: MessagesApi, appConfig: ApplicationConfig) extends FrontendErrorHandler with I18nSupport {
+
+  val logger: Logger = Logger(this.getClass)
+
+  override def standardErrorTemplate(pageTitle: String, heading: String, message: String)(implicit request: Request[_]): Html =
+    views.html.errors.error(pageTitle, heading, message)
+
+  override def internalServerErrorTemplate(implicit request: Request[_]): Html = {
+    views.html.errors.technicalDifficulties(extractErrorReference(request), getDateTime)
+  }
+
+  private def getDateTime: LocalDateTime = {
+    val instant = Instant.ofEpochMilli(System.currentTimeMillis)
+    LocalDateTime.ofInstant(instant, ZoneId.of("Europe/London"))
+  }
```

Comment:
is this different from LocalDateTime.now ?

Reply:
seems copy-paste from one of these places : https://github.com/search?q=org%3Ahmrc+%22++++val+instant+%3D+Instant.ofEpochMilli%28System.currentTimeMillis%29%22&type=Code

Reply:
This was copy and paste, I can change to LocalDateTime.now()